server/api/users/users.test.ts

```diff
@@ -37,19 +37,66 @@ describe('auth router', () => {
   });
 
   describe('GET /', () => {
-    it('expects empty array if no users exist', async () => {
+    it('expects empty array if no users exist on the homepage', async () => {
```

Comment:
API routes don't really have a relation to pages - a homepage may call an API route but an API route is often crafted independently of the page it will used on.  i think we can just remove this added string